### PR TITLE
No functional changes. Only Docstring changes and formatting. removed…

### PR DESCRIPTION
**Pull request by malcolm-dsider**
… doc-author and "param self"; added type and rtype.

### CODE REVIEW

File: tests/examples/Wanju_Yuan_Closed-Loop_Geothermal_Energy_Recovery.txt [hunk 1]
```diff
@@ -1,72 +1,203 @@
...
-Inflated Equity Interest Rate,0.12,	--- [-] Required if Bicycle model is selected. See manual for details.
-Inflation Rate,0.025,			--- [-] Required if Bicycle model is selected. See manual for details.
-Combined Income Tax Rate,0.392,		--- [-] Required if Bicycle model is selected. See manual for details.
-Gross Revenue Tax Rate,0,		--- [-] Required if Bicycle model is selected. See manual for details.
-Reservoir Stimulation Capital Cost,0,				--- [M$/injection well] Reservoir stimulation capital cost per injection well
-
-
-*** Simulation Parameters ***
-Print Output to Console,1,		--- [-] Should be 0 (don't print results to console) or 1 (print results to console)
+                               *****************
```
Comment (softwareengineerprogrammer) [inline tests/examples/Wanju_Yuan_Closed-Loop_Geothermal_Energy_Recovery.txt:1]: Seems like this was an accident? (input file content replaced with output content and output file deleted)

**Review by softwareengineerprogrammer — CHANGES_REQUESTED** [review]
Looks good - one issue with test files as commented

**Comment by softwareengineerprogrammer** [comment]
@malcolm-dsider To address the unintended changes to the Wanju files, you can do the following:

```
git co e3c6c4c79425976085a7d939e4e00a308146a1b4 -- tests/examples/Wanju_Yuan_Closed-Loop_Geothermal_Energy_Recovery.out tests/examples/Wanju_Yuan_Closed-Loop_Geothermal_Energy_Recovery.txt

git commit -m "Revert unintended changes to Wanju_Yuan_Closed-Loop_Geothermal_Energy_Recovery test files"
```

(And then `git push`)

**Comment by malcolm-dsider** [comment]
Jonathan – 

 

Thanks for your help on this – It ws an error to commit that  output file as an input file, and I wasn’t certain how to undo the error, so I appreciate the advice….

 

But the first command seems to fail. There is no “git co” command???

 

Thanks

 

-Malcolm

**Comment by malcolm-dsider** [comment]
Never mind – figured it out co=checkout

 

From: ***@***.*** ***@***.***> 
Sent: Friday, November 10, 2023 2:00 PM
To: 'NREL/python-geophires-x' ***@***.***>; 'NREL/python-geophires-x' ***@***.***>
Cc: 'Mention' ***@***.***>
Subject: RE: [NREL/python-geophires-x] No functional changes. Only Docstring changes and formatting. removed… (PR #45)

 

Jonathan – 

 

Thanks for your help on this – It ws an error to commit that  output file as an input file, and I wasn’t certain how to undo the error, so I appreciate the advice….

 

But the first command seems to fail. There is no “git co” command???

 

Thanks

 

-Malcolm

**Comment by softwareengineerprogrammer** [comment]
> There is no “git co” command???

Yes, my apologies. (I noticed that after I commented and edited hoping to avoid the confusion but seems I wasn't quick enough.)

`co` is a very common alias for `checkout`, and I forgot not everyone has it. You can configure it by running:

```
git config --global alias.co checkout
```

which I would recommend.

Glad you got it figured out!

**Review by softwareengineerprogrammer — COMMENTED** [review]
2nd revision looks good (retroactively approved post-merge for posterity's sake)

**Pull request merged**